src/traces/contourcarpet/index.js

```diff
@@ -15,7 +15,7 @@ ContourCarpet.supplyDefaults = require('./defaults');
 ContourCarpet.colorbar = require('../contour/colorbar');
 ContourCarpet.calc = require('./calc');
 ContourCarpet.plot = require('./plot');
-ContourCarpet.style = require('./style');
```

Comment:
This is great. I'd have to dig in further to refresh my memory, but I struggled a bit to wrangle the behavior under control for both constraints and contours since it felt like they had slightly different semantics. Like constraints tend to be a single color while contours have multiple colors. But if it passes the existing image tests and fixes other behaviors, then I'm 100% content.